cfgov/v1/forms.py

```diff
@@ -262,6 +269,7 @@ def get_query_strings(self):
             'categories__name__in',  # categories
             'tags__slug__in',        # topics
             'authors__slug__in',     # authors
+            'statuses__status__in',  # statuses
```

Comment:
I'm trying to think of a situation where this-- having a different number of query parameters for this form class-- might cause a problem. Maybe if the `status` checkbox is checked on a form that's not an EnforcementActionsFilterPage? I haven't dug into it.

Reply:
Well the `EnforcementActionsFilterForm` is only used on instances of `EnforcementActionsFilterPage`, so I think this is safe.